### ISSUE TITLE
How to use the generic json updater of release please github action

### ISSUE THREAD

**Issue by danielo515** [google-github-actions/release-please-action#727]
### TL;DR

I have some non standard projects with a json file to update with versions. 

### Detailed design

Release please looks like a nice github action for lots different kinds of projects. I have some projects that are not in any of the official supported ones, but my requirements are very simple. Bump a version in a json file and execute a publish command. There seem to be a generic json updater, but there are no instructions on how to use it in the github action. What will we the way to configure the action to use that specific updater?

Thanks 

### Additional information

Thank you for this action that greatly simplifies the process of automatic releases
